metasource/driver/pull.go

```diff
@@ -45,27 +45,27 @@ func DownloadRepositories(unit *home.FileUnit, vers *string, stab int64, cast *i
 	if expt != nil {
 		stab += 1
 		slog.Log(nil, slog.LevelDebug, fmt.Sprintf("[%s] Stab failed for %s due to %s", *vers, unit.Name, expt.Error()))
-		return DownloadRepositories(unit, vers, stab, cast)
+		return DownloadRepositories(unit, vers, stab, cast, loca)
 	}
 
 	resp, expt = oper.Do(rqst)
 	if expt != nil {
 		stab += 1
 		slog.Log(nil, slog.LevelDebug, fmt.Sprintf("[%s] Stab failed for %s due to %s", *vers, unit.Name, expt.Error()))
-		return DownloadRepositories(unit, vers, stab, cast)
+		return DownloadRepositories(unit, vers, stab, cast, loca)
```

Comment:
Here again, please use the `DBFOLDER` variable that has been set already in the global variable here https://github.com/gridhead/metasource/blob/main/main.go#L26 instead of passing `loca` here as a parameter.